### PR TITLE
[SW-3] Set-up React Router

### PR DESCRIPTION
**Pull request by svekukucska**
* added React Router DOM to set-up

### CODE REVIEW

**Review by spinenco-sv — APPROVED** [review]
🎣

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,22 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends Component {
```
Comment (svIZsollei) [inline src/App.js:7]: Can we use functional component here?

**Review by svIZsollei — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 2]
```diff
@@ -328,3 +328,6 @@ ASALocalRun/
 
 # MFractors (Xamarin productivity tool) working folder 
 .mfractor/
+
+# VS Code
+.vscode/
```
Comment (svekozak) [inline .gitignore]: @svekukucska please, add a new line

File: src/App.js (hunk 1 above)
Comment (svekukucska) [reply src/App.js:7]: I would rather keep it a class component, as I don't know yet, what exactly will be included here in the future and if we will need/use state or not.

File: src/pages/Home.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
```
Comment (svekozak) [inline src/pages/Home.js]: Please, make a folder for this file named "Home" and rename the file index.js

File: src/pages/NotFound.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
```
Comment (svekozak) [inline src/pages/NotFound.js]: Please, make a folder for this file named "NotFound" and rename the file index.js

**Review by svekozak — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (svekukucska) [reply .gitignore]: I will implement the requested changes and amend it to this PR.

File: src/pages/Home.js (hunk 3 above)
Comment (svekukucska) [reply src/pages/Home.js]: I will implement the requested changes and amend it to this PR.

File: src/pages/NotFound.js (hunk 4 above)
Comment (svekukucska) [reply src/pages/NotFound.js]: I will implement the requested changes and amend it to this PR.

**Review by svekozak — APPROVED** [review]

**Pull request merged**